cmd/yggd/main.go

```diff
@@ -146,13 +146,23 @@ func main() {
 			Server:         c.String(cliServer),
 			CertFile:       c.String(cliCertFile),
 			KeyFile:        c.String(cliKeyFile),
-			CaRoot:         c.String(cliCaRoot),
+			CARoot:         c.StringSlice(cliCaRoot),
 			TopicPrefix:    c.String(cliTopicPrefix),
 			Protocol:       c.String(cliProtocol),
 			DataHost:       c.String(cliDataHost),
 			ExcludeWorkers: map[string]bool{},
 		}
 
+		tlsConfig, err := DefaultConfig.CreateTLSConfig()
+		if err != nil {
+			return cli.Exit(fmt.Errorf("cannot create TLS config: %w", err), 1)
+		}
+
+		TlSEvents, err := DefaultConfig.WatcherUpdate()
+		if err != nil {
+			return cli.Exit(fmt.Errorf("cannot started TLS config: %w", err), 1)
```

Comment:
I think this is meant to be something like "cannot start watching for certificate changes"?